ta4j-core/src/test/java/org/ta4j/core/aggregator/DurationBarAggregatorTest.java

```diff
@@ -115,4 +115,16 @@ public void upscaledTo10DayBars() {
         TestUtils.assertNumEquals(num1.numOf(293), bar1.getVolume());
     }
 
+    /**
+     * Tests if the bars are upscaled correctly from 1day to 10day, allowed not final bars too
+     */
+    @Test
+    public void upscaledTo10DayBarsNotOnlyFinalBars() {
+        final DurationBarAggregator barAggregator = new DurationBarAggregator(Duration.ofDays(10), false);
+        final List<Bar> bars = barAggregator.aggregate(getOneDayBars());
+
+        // must be 1 bars
```

Comment:
please change comment to `// must be 2 bars`

Reply:
Fixed